src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -1,35 +1,83 @@
 package com.kodilla.ecommercee.controller;
-
+import com.kodilla.ecommercee.dao.OrderDao;
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.OrderItem;
 import com.kodilla.ecommercee.domain.dto.CartDto;
 import com.kodilla.ecommercee.domain.dto.ProductDto;
+import com.kodilla.ecommercee.exception.NotFoundException;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import com.kodilla.ecommercee.mapper.ProductMapper;
+import com.kodilla.ecommercee.service.CartService;
+import com.kodilla.ecommercee.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
+
 import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
+@CrossOrigin(origins = "*")
 @RestController
 @RequestMapping("api/v1/ecommercee/carts")
 public class CartController {
+    @Autowired
+    CartService cartService;
+
+    @Autowired
+    ProductService productService;
+
+    @Autowired
+    CartMapper cartMapper;
+
+    @Autowired
+    ProductMapper productMapper;
+
+    @Autowired
+    OrderDao orderRepository;
+
     @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     public void newCart(@RequestBody CartDto cartDto) {
```

Comment:
make naming more consistent
```suggestion
    public void add(@RequestBody CartDto cartDto) {
```